src/test/java/seedu/address/logic/commands/general/CommandTestUtil.java

```diff
@@ -102,6 +108,65 @@
+    public static final String VALID_END_DATE_TIME_TUTORIAL = "17-10-2021 16:00";
+    public static final String VALID_START_DATE_TIME_EXAM = "20-10-2021 09:00";
+    public static final String VALID_END_DATE_TIME_EXAM = "20-10-2021 11:00";
+    public static final String VALID_DESCRIPTION_TUTORIAL = "Topics: Drawing UML diagrams";
+    public static final String VALID_DESCRIPTION_EXAM = "I'm very unprepared";
+    public static final String VALID_ADDRESS_TUTORIAL = "2-11 COM2";
+    public static final String VALID_ADDRESS_EXAM = "Zoom";
+    public static final String VALID_ZOOM_TUTORIAL = "https://nus-sg.zoom.us/j/0123456789?pwd=ABCDEFG";
+    public static final String VALID_ZOOM_EXAM = "nus-sg.edu/123%a";
+    public static final String VALID_TAG_COOL = "cool";
+    public static final String VALID_TAG_EXAMS = "exams";
+
+    public static final String NAME_DESC_TUTORIAL = " " + PREFIX_NAME + VALID_NAME_TUTORIAL;
+    public static final String NAME_DESC_EXAM = " " + PREFIX_NAME + VALID_NAME_EXAM;
+    public static final String START_DATE_TIME_DESC_TUTORIAL = " " + PREFIX_START_TIME + VALID_START_DATE_TIME_TUTORIAL;
+    public static final String START_DATE_TIME_DESC_EXAM = " " + PREFIX_START_TIME + VALID_START_DATE_TIME_EXAM;
+    public static final String END_DATE_TIME_DESC_TUTORIAL = " " + PREFIX_END_TIME + VALID_END_DATE_TIME_TUTORIAL;
+    public static final String END_DATE_TIME_DESC_EXAM = " " + PREFIX_END_TIME + VALID_END_DATE_TIME_EXAM;
+    public static final String DESCRIPTION_DESC_TUTORIAL = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_TUTORIAL;
+    public static final String DESCRIPTION_DESC_EXAM = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_EXAM;
+    public static final String ADDRESS_DESC_TUTORIAL = " " + PREFIX_ADDRESS + VALID_ADDRESS_TUTORIAL;
+    public static final String ADDRESS_DESC_EXAM = " " + PREFIX_ADDRESS + VALID_ADDRESS_EXAM;
+    public static final String ZOOM_DESC_TUTORIAL = " " + PREFIX_ZOOM + VALID_ZOOM_TUTORIAL;
+    public static final String ZOOM_DESC_EXAM = " " + PREFIX_ZOOM + VALID_ZOOM_EXAM;
+    public static final String TAG_DESC_COOL = " " + PREFIX_TAG + VALID_TAG_COOL;
+    public static final String TAG_DESC_EXAMS = " " + PREFIX_TAG + VALID_TAG_EXAMS;
+    public static final String TAG_DESC_DELETEEXAMS = " " + PREFIX_DELETE_TAG + VALID_TAG_EXAMS;
+    public static final String TAG_DESC_DELETECOOL = " " + PREFIX_DELETE_TAG + VALID_TAG_COOL;
+
+    public static final String EMPTY_PREFIX_START_DATE_TIME = " " + PREFIX_START_TIME;
+    public static final String EMPTY_PREFIX_END_DATE_TIME = " " + PREFIX_END_TIME;
+    public static final String EMPTY_PREFIX_DESCRIPTION = " " + PREFIX_DESCRIPTION;
+
+    public static final String INVALID_EVENT_NAME_DESC = " " + PREFIX_NAME + "Hackathon&"; // '&' not allowed in names
+    public static final String INVALID_START_DATE_TIME_DESC = " " + PREFIX_START_TIME + "911a";
+    // 'a' not allowed in phones
```

Comment:
The comments for INVALID_START_DATE_TIME_DESC and INVALID_END_DATE_TIME_DESC should be date and time only takes in dd-MM-yyy HH:mm format